python/ray/experimental/state/common.py

```diff
@@ -368,23 +358,23 @@ class ActorState(StateSchema):
     #: If the actor is restarting, it could be the node id
     #: of the dead actor (and it will be re-updated when
     #: the actor is successfully restarted).
-    node_id: str = state_column(filterable=True)
+    node_id: Optional[str] = state_column(filterable=True)
```

Comment:
Changes are mostly due to:
1. Add Optional because some fields could be None, and pydantic checks it. 
2. Reorder because optional fields have to be after non-optional.